### DISCUSSION TITLE
Recommended way to keep in sync with fwupd releases

### DISCUSSION THREAD

**Discussion by erazemk** [fwupd/fwupd#4974] [Q&A] [answered]
In the latest blog post you say: *"If you want to see both new bits of data then make sure you’re using gnome-control-center from the main branch and then install the new fwupd version – although if you’re stuck on a distro version of fwupd GNOME will still fallback to the single-line summary line as before."*

Just out of curiosity, what's the recommended way to keep an up-to-date gnome-control-center (Maybe flatpak in gnome-nightly or constantly compiling from main branch, or Fedora Rawhide?) and more importantly, what's the recommended way to keep up with fwupd main (as opposed to using a "distro version")?

**Comment by hughsie** [comment, marked as answer]
For fwupd I normally recommend the Fedora COPR; that's at least been tested by me: https://copr.fedorainfracloud.org/coprs/rhughes/fwupd/ -- but there's also a snap of the daily code that's autogenerated an untested. GNOME Control Center is harder, as it's part of core gnome and probably depends on the same cutting edge versions of gsettings-desktop-schema and gnome-settings-daemon. For that I'd recommend installing the latest Fedora test release, although that's only going to have the latest tarball release -- which could be quite a lot older than the `main` branch.

**Reply by erazemk** [reply]
Thanks :)
